build.gradle

```diff
@@ -4166,7 +4166,15 @@ task javadoc(type: Javadoc, dependsOn: createMSPfile) {
     dependsOn(projectsToDocument.collect { project -> project.getTasksByName("classes", true)});
 }
 
+task initDirs() {
+    // JDK-8290473
+    file("modules/javafx.graphics/build/hlsl/Decora").mkdirs()
+    file("modules/javafx.graphics/build/hlsl/Prism").mkdirs()
+}
+
 task sdk() {
+    dependsOn(initDirs)
```

Comment:
the reason I put them at the top level is to avoid a situation when gradle thinks that the graphics task is up-to-date (ex.: after a branch change) but the directories are not there.
but you are right, this is a better place to add.  thank you.